dev/io.openliberty.transport.http_fat/fat/src/io/openliberty/transport/http_fat/accesslists/Utils.java

```diff
@@ -0,0 +1,320 @@
+            testMethod = testMethod.replace(RepeatTestFilter.getRepeatActionsAsString(), "");
+        }
+        String serverXml = testMethod + "_server.xml";
+        String customized = customize(server, serverXml, test.clientDetails);
+        try {
+            // We will wait for the app to come up
+            replaceServerXmlContents(server, customized, test.app, READY);
+        } catch (Exception e) {
+            debug(e.getMessage());
+            Assert.fail(e.getMessage());
+        }
+    }
+
+    /**
+     * Replace a servers server.xml contents with the contents of a String
+     *
+     * @param serverXml the new config (contents not filename)
+     * @throws Exception
+     */
+    static void replaceServerXmlContents(LibertyServer server, String serverXml, String app, String... additionalMsgs) throws Exception {
+        RemoteFile srvXml;
+        try {
+            srvXml = server.getServerConfigurationFile();
+            debug("Replacing server.xml for server " + server.getServerName() + " with: ");
+            debug(serverXml);
+            server.setMarkToEndOfLog();
+            Files.write(Paths.get(srvXml.getAbsolutePath()), serverXml.getBytes(), StandardOpenOption.TRUNCATE_EXISTING);
+            server.waitForConfigUpdateInLogUsingMark(Collections.singleton(app), additionalMsgs);
+
+            if (serverXml.contains("tcpOptions")) {
+                ServerConfiguration c = server.getServerConfiguration();
+                ConfigElementList<HttpEndpoint> h = c.getHttpEndpoints();
+                for (HttpEndpoint httpEndpoint : h) {
+                    TcpOptions tcpo = httpEndpoint.getTcpOptions();
+                    debug("TcpOptions have been set to be: " + tcpo);
+                }
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+            debug(e.getMessage());
+            throw e;
+        }
+    }
+
+    /**
+     * Complete a template with property values
+     *
+     * @param clientInfoServer
+     * @param serverXml
+     */
+    static String customize(LibertyServer server, String file, Properties p) {
+        String custom = "empty";
+        String src = server.pathToAutoFVTTestFiles + file;
+
+        try (Stream<String> strm = new BufferedReader(new FileReader(new File(src))).lines()) {
+            custom = strm.collect(Collectors.joining("\n"));
+            final StringBuilder sb = new StringBuilder(custom);
+            // Substitute any keys for values from Properties - short and simple (prefixes etc. not needed)
+            if (p != null) {
+                p.forEach((k, v) -> sb.replace(0, sb.length(), sb.toString().replaceAll((String) k, (String) v)));
+            }
+            custom = sb.toString();
+            return custom;
+        } catch (IOException e) {
+            e.printStackTrace();
+            Assert.fail("failed to cusotmize " + src + " from " + p + " due to " + e.getMessage());
```

Comment:
extremely minor: `cusotmize` -> `customize`

Reply:
Apologies and thanks for spotting it.